certified-connectors/Litera/apiDefinition.swagger.json

```diff
@@ -0,0 +1,198 @@
+		"contact": {
+			"name": "Litera Customer Care",
+			"url": "https://www.litera.com/support",
+			"email": "support@litera.com"
+		},
+		"x-ms-api-annotation": {
+			"status": "Preview"
+		},
+		"x-ms-keywords": [
+			"Litera",
+			"GetMatterList",
+			"GetMatterNarrative",
+			"GetMatterNarratives",
+			"GetMatterDetail",
+			"GetMatterDetails",
+			"matter list",
+			"list of matters",
+			"matters",
+			"matter",
+			"deals",
+			"deal",
+			"cases",
+			"case",
+			"legal matters",
+			"legal matter",
+			"legal cases",
+			"legal case",
+			"legal deals",
+			"legal deal",
+			"transactions",
+			"transaction",
+			"matter narrative",
+			"matter overview",
+			"matter description",
+			"description for matter",
+			"narrative for matter",
+			"overview for matter",
+			"description of matter",
+			"narrative of matter",
+			"overview of matter"
+		]
+	},
+	"host": "copilotdevappservice.azurewebsites.net",
+	"basePath": "/",
+	"schemes": [
+		"https"
+	],
+	"consumes": [],
+	"produces": [],
+	"paths": {
+		"/GetMatterList": {
+			"get": {
+				"tags": [
+					"Matter List"
+				],
+				"responses": {
+					"200": {
+						"description": "default",
+						"schema": {}
+					}
+				},
+				"produces": [
+					"application/json"
+				],
+				"summary": "Allows the user to search for legal matters and their associated IDs.",
+				"description": "Allows the user to search for legal matters (also known as deals or transactions) and their associated IDs. The endpoint returns a list of matter IDs and their corresponding matter names which satisfy the given natural language request. The request can contain search criteria involving matter fields such as matter location, industry, client name, etc. Example requests include \\\"Show me New York matters\\\", \\\"Get me matters for clients in the financial industry\\\", \\\"Find me deals in tech\\\".",
+				"operationId": "GetMatterList",
+				"parameters": [
+					{
+						"name": "request",
+						"description": "Natural language search request for legal clauses or definitions",
+						"in": "query",
+						"type": "string",
+						"x-ms-summary": "Natural language search request for legal clauses or definitions",
+						"required": true
+					}
+				]
+			}
+		},
+		"/GetMatterNarrative": {
+			"get": {
+				"tags": [
+					"Matter Narrative"
+				],
+				"produces": [
+					"application/json"
+				],
+				"responses": {
+					"200": {
+						"description": "default",
+						"schema": {}
+					}
+				},
+				"parameters": [
+					{
+						"name": "matterId",
+						"description": "The matter ID (matter number) of the matter for which to retrieve the narrative.",
+						"in": "query",
+						"type": "string",
+						"required": true,
+						"x-ms-summary": "The matter ID (matter number) of the matter for which to retrieve the narrative."
+					}
+				],
+				"summary": "Allows the user to retrieve a matter narrative for the specified legal matter.",
```

Comment:
'summary' value must end with alphanumeric character. ValuePath: GetMatterNarrative/get/summary

Reply:
done - will be reflected in next commit.